tools/analyzer_plugin/README.md

```diff
@@ -97,7 +138,45 @@ Congrats, you're debugging! 🎉
 
 You can now set breakpoints, view logs, and do everything else you'd normally do in the debugger for.
 
+## Feature Ideas & Inspiration
+
+We drew inspiration from the following:
+* Flutter analysis functionality (actually built into the Dart SDK via [analysis_server][analysis_server]).
+* React JS IDE plugins
+    * [ESLint Plugin](https://github.com/yannickcr/eslint-plugin-react)
+    * [ReactEd](https://marketplace.visualstudio.com/items?itemName=ReactEd.reacted)
+* [AngularDart analyzer plugin](https://github.com/dart-lang/angular/tree/master/angular_analyzer_plugin)
+
+#### Abandoned Feature Ideas
+
+* Outline for similar behavior to closing tags.
+    * Outline implementation is barely supported in analyzer plugin.
+    * Need to suggest closing tags API for plugin, maybe this would be a good entry point.
+* Breadcrumbs: part of the IDE plugin, not the analyzer plugin.
+    * Greg has most of a JetBrains plugin done.
+* [Prop drilling](https://marketplace.visualstudio.com/items?itemName=ReactEd.reacted#prop-drilling): isn't supported by the plugin.
+
+## Limitations
+
+### Dart `analyzer_plugin` Limitations
+
+* Not advertised for public use (but [public use isn't discouraged](https://github.com/dart-lang/sdk/issues/35516)).
+* API is pretty rough in some spots.
+* Outline can't be empty, seems to only work intermittently work (at least in IntelliJ).
+* Assist/Fix Mixin APIs not async, preventing usage of edit builders (which are recommended); we have our own local copies of these that fix this, and should probably be contributed back at some point.
+* Some features might not be supported by all IDEs.
+* Workflow involves restarting the analysis server every time you make a change.
+* At times, lints do not resolve in the IDE unless you restart the analysis server.
+
+### Analyzer Limitations
+
+* Can easily go from AST to Elements but not the other way around (`computeNode` is deprecated). Most of the time should not be problematic since if you have access to an Element, you usually have access to the corresponding AST.
+    * Might be able to work around this using `NodeLocator`.
+* `toSource` should be used with caution since it's an approximation of the source.
+
```

Comment:
Many of these items are either out of date, will be captured by issues, or aren't written with interneal audiences in mind. Can we remove these sections?